.github/workflows/cmake.yml

```diff
@@ -0,0 +1,90 @@
+# inspired by "build GDAL library and to run test with CMake"
+# from https://github.com/OSGeo/gdal/blob/master/.github/workflows/cmake_builds.yml
+
+---
+name: CMake
+
+on:
+    push:
+        paths-ignore:
+            - 'doc/**'
+        branches:
+            - main
+            - releasebranch_*
+    pull_request:
+        paths-ignore:
+            - 'doc/**'
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+    cancel-in-progress: true
+
+env:
+    CMAKE_UNITY_BUILD: OFF
+
+permissions:
+    contents: read  #  to fetch code (actions/checkout)
+
+jobs:
+
+    build-cmake:
+        runs-on: ubuntu-22.04
+        env:
+            PythonVersion: 3.9
```

Comment:
Would it make sense to start with latest versions, 24 and 3.13?

Reply:
For Ubuntu 24, does the package list we currently have work correctly? If not, that's something to handle when upgrading everything else to 24.04, not for the first iteration. We all want to merge this soon

Reply:
I didn't realize that. Then 22 by all means.

Reply:
For 3.13, there's #4372, problems with doctests